coralproject/talk
# Added .nvmrc file

## What does this PR do?

- Locked node version at 8 via `.nvmrc` file

## How do I test this PR?

- Install and enable [nvm](https://github.com/creationix/nvm#installation)
- Run `nvm use` in the Talk directory
- Notice it adopt the specified `8.*` node version
